lib/imgix/path.rb

```diff
@@ -137,6 +137,11 @@ def build_srcset_pairs(width_tolerance:, params:)
         widths = @target_widths
       end
 
+      if min_width != MIN_WIDTH || max_width != MAX_WIDTH
+        validate_range!(min_width,max_width)
+        widths = widths.select { |w| min_width <= w && w <= max_width } 
+      end
+
```

Comment:
Looks like we're making our full list first, then truncating it based on the min and max. Why not start with the min and max and build the list of widths based on that? It would save us a step, and in the case of an extremely low max (or an extremely high min) it would save us a lot of loops when building the width list.

Reply:
This approach would also have the benefit of actually including the given min and max widths in the list, which is probably what the user would expect to see.

Reply:
Good point, they should definitely be included